pages/form/3.js

```diff
@@ -26,6 +27,7 @@ export default function Question3() {
   if (travelDays.length < 1) {
     return (
       <Layout>
+        <Q3Progress />
```

Comment:
I think this section requires a bit of refactoring.
the `<Q3Progress />` component needs to be moved outside of the `if` block, but remain within `<Layout />` block.

Lemme know if you need any help / suggestions with this.

Reply:
Should the progress bar appear on the page regardless of which path the app takes @hqtan?

Reply:
Yups! progress bar should appear regardless of which path the form takes 👍

Reply:
Ok cool. I think it needs to be in both the if statement and the other return. I have changed that my end and will update.